src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -2883,6 +2888,8 @@ class C : I
             Assert.Equal("I.set_P", explicitPropertySetterSymbol.Name);
             Assert.Equal(1, explicitPropertySetterSymbol.ExplicitInterfaceImplementations.Length);
             Assert.Same(explicitPropertySymbol.SetMethod, explicitPropertySetterSymbol);
+
+            Assert.NotNull(explicitPropertySymbol.BackingField);
```

Comment:
>Assert.NotNull(explicitPropertySymbol.BackingField); [](start = 12, length = 52)

I think we need a set of pure symbol tests (not involving SemanticModel at all) for the new APIs, which should cover the breadth of scenarios: declared in source, declared in metadata, substituted, retargeting, etc.